File: Codigo/Obligatorio1DA2/Domain/Category.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Category
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public List<TouristicPointsCategory> TouristicPoints { get; set; }

        public Category()
        {

        }
        public Category (string name)
        {
            Name = name;
        }

        public override bool Equals(object obj)
        {
            if (obj == null)
            {
                return false;
            }
            if (!(obj is Category))
            {
                return false;
            }
            return Name == ((Category)obj).Name;
        }
    }
}

File: Codigo/UruguayNatural/src/Models/Category.ts
export class Category {
    id: number;
    name: string;
    touristicPoints : string[];
  
    constructor(id: number, name: string, touristicPoints: string[]) {
      this.id = id;
      this.name = name;
      this.touristicPoints = touristicPoints;
  }

}